### DISCUSSION TITLE
The Future of DungeonRevealer

### DISCUSSION THREAD

**Discussion by TheMBeat** [dungeon-revealer/dungeon-revealer#1658] [General]
Hello @maxb2 and @n1ru4l is DR dead?

**Comment by dereknola** [comment]
I got tired of trying to use unmaintained forks that ultimately proved too slow to run on my android TV, so I made https://github.com/dereknola/FogCast as a lightweight replacement.

**Reply by maxb2** [reply]
That looks like a good alternative! I started a rewrite of dungeon-revealer here:
https://github.com/maxb2/dungeon-revealer-go

It's kind of ugly, but it does have dynamic lighting.
